resources/lang/de.json

```diff
@@ -377,7 +377,7 @@
     "settings.prevent": "Verbieten",
     "settings.privacy.update.success": "Deine Privatsphäreneinstellungen wurden aktualisiert.",
     "settings.search-engines.description": "Wir setzen ein noindex-Tag auf deinem Profil um den Suchmaschinen mitzuteilen, dass dein Profil nicht indexiert werden darf. Wir haben keinen Einfluss darauf, ob die Suchmaschine diesen Wunsch berücksichtigt.",
-    "settings.no-tokens": "Es haben aktuell keine externen Anwendungen Zugriff auf deinen Account.",
+    "settings.no-access-from-external-applications": "Es haben aktuell keine externen Anwendungen Zugriff auf deinen Account.",
```

Comment:
Few things: If you change a key in the language file, please make sure that it isn't used elsewhere.

For the `no-tokens` key in particular, it's used in `resources/views/settings/cards/api-token.blade.php` where it makes sense as is.

Also, just because an application has not set a Webhook, it doesn't mean that it doesn't have access to the account, TRWL just does not push status updates to it.

In this case, I'd suggest duplicating the line and adding a `settings.no-webhooks` key, which could say something like "Es werden keine externen Anwendungen über Änderungen an deinem Account informiert". This way, no existing behaviour is modified.

Reply:
I used IntelliJ's rename feature here hoping it would rename everything, but your suggestion is probably way better with a more expressive description.

Reply:
Fixed with 72936a54386226f5a468be1ec97126e046ee1547.

Thaaankssss